kafka_topic_enforcer/src/main/java/com/tesla/data/topic/enforcer/ConfigDrift.java

```diff
@@ -15,27 +15,30 @@
  */
 public class ConfigDrift {
 
-  // permissible % increase threshold doesnt apply if the increased amount is below low water mark
+  // permissible % increase threshold doesnt apply if the increased amount is
+  // below low water mark
```

Comment:
it makes reviewing much easier if you don't change unrelated lines like these. In fact, changing these lines looks wrong - we should be supporting a 120 wide line

Reply:
yeah this is my bad, didn't have my IDE properly configured style-wise and the auto-formatter wreaked a bit of havoc. correcting this now, will get the unrelated formatting changes out.